tests/jenkins/TestDataPrepperDistributionArtifacts.groovy

```diff
@@ -3,35 +3,55 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import jenkins.tests.BaseJobPipelineTest
+
+import jenkins.tests.BuildPipelineTest
 import org.junit.Before
 import org.junit.Test
 
-class TestDataPrepperDistributionArtifacts extends BaseJobPipelineTest {
+class TestDataPrepperDistributionArtifacts extends BuildPipelineTest {
 
     @Before
     void setUp() {
         super.setUp()
 
+        // Variables for Data-prepper-Distribution-Artifacts-job
+        def workspace = '/tmp/workspace'
         binding.setVariable('VERSION', '0.22.1')
         binding.setVariable('BRANCH', 'refs/tags/0.22.1')
-
         binding.setVariable('JOB_BASE_NAME', 'data-prepper-distribution-artifacts')
-        binding.setVariable('BUILD_NUMBER', '7654321')
-        binding.setVariable('WORKSPACE', '/tmp/workspace')
-
+        binding.setVariable('BUILD_NUMBER', '51')
+        binding.setVariable('WORKSPACE', workspace)
         binding.setVariable('ARTIFACT_BUCKET_NAME', 'job-s3-bucket-name')
 
         helper.registerAllowedMethod('checkout', [Map], {})
-        helper.registerAllowedMethod('signArtifacts', [Map], {})
-        helper.registerAllowedMethod('uploadToS3', [Map], {})
-        helper.registerAllowedMethod('printArtifactDownloadUrlsForStaging', [Map], {})
-        helper.registerAllowedMethod('postCleanup', [], {})
+
+        def artifactsPath = "$workspace/release/archives/linux/build/distributions"
+        helper.addShMock("find ${artifactsPath} | sed -n \"s|^${artifactsPath}/||p\"") { script ->
+            return [stdout: "tar_dummy_1_artifact_1.0.0.tar.gz tar_dummy_2_artifact_1.0.0.tar.gz", exitValue: 0]
+        }
+
+        // Variables for signArtifacts library
+        TestSignArtifacts.setUpVariables(binding, helper)
+
+        // Upload to s3
+        TestUploadToS3.setUpVariables(binding, helper)
+
+        // Post cleanup
+        helper.registerAllowedMethod("cleanWs", [Map])
     }
 
     @Test
     void dataPrepperDistributionArtifacts_builds_consistently() {
-        testPipeline('jenkins/data-prepper/distribution-artifacts.jenkinsfile',
-                'tests/jenkins/jobs/data-prepper/distribution-artifacts.jenkinsfile')
+        super.testPipeline('jenkins/data-prepper/distribution-artifacts.jenkinsfile',
+                'tests/jenkins/jenkinsjob-regression-files/data-prepper/distribution-artifacts.jenkinsfile')
+
+        TestSignArtifacts.verifySignArtifactsParams(helper, ".sig", "linux",
+                "/tmp/workspace/release/archives/linux/build/distributions")
+
+        def filenamesForUrls = ['tar_dummy_1_artifact_1.0.0.tar.gz', 'tar_dummy_1_artifact_1.0.0.tar.gz.sig',
```

Comment:
This is a similar comment to the one above. What are these string names? Do they need to be these exact values?

Reply:
These are dummy names for the artifacts that gets created with the job.